src/scripts/core/models/background.ts

```diff
@@ -1,92 +1,175 @@
-        this._farWave.y = this._waveStartingPosition;
+        this._farWave.y = this._waveStartingPosition + this._waveStartingPosition * 0.2;
         this.initWave(this._midWave, this._viewWidth * 1.5, this._viewHeight);
         this._midWave.y = this._waveStartingPosition;
         this.initWave(this._closeWave, this._viewWidth * 2, this._viewHeight);
-        this._closeWave.y = this._waveStartingPosition;
+        this._closeWave.y = this._waveStartingPosition + this._waveStartingPosition * 0.5;
 
-        this._ship = new Token({
-            availHeight: this._viewWidth,
-            availWidth: this._viewHeight,
-            skIndex: 1
-        });
-        this._ship.scale.set(1);
-        this._ship.position = {
-            x: this._viewWidth * 0.9 ,
-            y: this._viewHeight
-        }
-        this._ship.pivot = {
-            x: this._ship.width * 0.5, 
-            y: this._ship.height * 0.25
-        }
-        this._ship.angle = -20;
-        this._ship.alpha = 0;
-        gsap.to(this._ship, {
-            duration: 5,
-            repeat: -1,
-            yoyo: true,
-
-            y: this._ship.y + this._ship.height * 0.1,
-            angle: 20,
-        });
+        this._npcShipClose = this.addShip(0.4, 9);
+        this._npcShipClose.x = this._viewWidth * 0.3;
+        this._playerShip = this.addShip(0.6, 5);
+        this._playerShip.y = this._viewHeight - this._playerShip.height * 0.4
+        this._npcShipFar = this.addShip(0.3, 3);
+        
 
-        this._midWaveContainer.addChild(this._ship, this._midWave);
+        this._midWaveContainer.addChild(
+            this._npcShipFar,
+            this._playerShip,
+            this._npcShipClose,
+            this._midWave
+        );
 
         const background = new Sprite(Assets.get('background'));
         background.width = this._viewWidth;
         background.height = this._viewHeight;
         
-        this.addWavesToTimeline(0.2, 0.2, 0.2, 2);
+        this.addWavesToTimeline(0.15, 0.1, 0, 2);
         this.addChild(
-            background,
+            this._stormCloudsContainer,
             this._farWave,
             this._midWaveContainer,
-            this._closeWave
+            this._closeWave,
+            ground
         );
+
+        this.stopThunder();
+    }
+
+    public playThunder(): void {
+        this._thunderFlash.alpha = 1;
+        this._lightning.alpha = 1;
+        this._thunderFlash.state.setAnimation(0, "introduction", false);
+        this._lightning.state.setAnimation(0, "introduction", false);
+    }
+
+    public loopThunder(): void {
+        this._thunderFlash.alpha = 1;
+        this._lightning.alpha = 1;
+        this._thunderFlash.state.setAnimation(0, "introduction", true);
+        this._lightning.state.setAnimation(0, "introduction", true);
+    }
+
+    public stopThunder(): void {
+        this._thunderFlash.alpha = 0;
+        this._lightning.alpha = 0;
```

Comment:
In pixi there is actually a strange bug with setting the alpha like this, where animations may play strangely when setting the alpha from 0 to 1. In the past we've actually set the alpha to 0.001 instead, which is functionally the same for all intents and purposes. But bypasses these bugs, obviously if there are no weird issues with your animations, don't do this. But it is worth keeping in mind

Said issue also happens if you set the 'renderable' property from false to true as well